nodes/obstacle_visualizer.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python3
+
+from typing import List, Dict
+
+import rospy
+
+from std_msgs.msg import Header
+from geometry_msgs.msg import Pose
+from tf.transformations import quaternion_from_euler
+from geometry_msgs.msg import Point, Quaternion, Vector3
+from visualization_msgs.msg import Marker, MarkerArray
+from std_msgs.msg import ColorRGBA, Header
+from SDFGenerator import SDFGenerator
+
+from gazebo_msgs.srv import SpawnModel, SpawnModelRequest
```

Comment:
```suggestion
from geometry_msgs.msg import Point, Quaternion, Vector3
from gazebo_msgs.srv import SpawnModel, SpawnModelRequest
from visualization_msgs.msg import Marker, MarkerArray
from std_msgs.msg import ColorRGBA, Header

from SDFGenerator import SDFGenerator
```
This order of includes make more sense